R/rank_regression.R

```diff
@@ -1,67 +1,128 @@
 #' Rank Regression for Parametric Lifetime Distributions
 #'
+#' @description
 #' This method fits an \strong{x on y} regression to a linearized
 #' two- or three-parameter cdf and is applicable for complete and (multiple) right
 #' censored data. The parameters are estimated in the frequently used (log-) location-scale
 #' parameterization. For the Weibull, estimates are transformed such that
 #' they are in line with the parameterization provided by the \emph{stats} package
-#' like \code{\link{pweibull}}.
+#' (see \code{\link{pweibull}}).
```

Comment:
Are you sure this link is functional? You might need \code{\link[stats]{pweibull}}